go/interpreter/lfvm/stack.go

```diff
@@ -15,12 +15,13 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/ethereum/go-ethereum/params"
 	"github.com/holiman/uint256"
 )
 
 var staticStackBoundry = [NUM_OPCODES]InstructionStack{}
 
+const StackLimit uint64 = 1024 // Maximum size of VM stack allowed.
```

Comment:
Should be private until needed to be public.

Reply:
I was actually thinking of making it a tosca constant that can be used by lfvm, ct and wherever else needed. But maybe that should a separate PR. For now I will just make it private